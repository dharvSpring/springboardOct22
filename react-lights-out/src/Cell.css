.Cell {
  height: 100px;
  width: 100px;
  background-color: darkslategray;
  cursor: pointer;
}

.Cell-lit {
  background-color: aliceblue;
}