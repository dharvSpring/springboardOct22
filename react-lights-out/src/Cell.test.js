import React from "react";
import { render } from "@testing-library/react";
import Cell from "./Cell";

describe("Cell comp tests", function() {
    let row;

    beforeEach(function() {
        const tr = document.createElement("tr");
        row = document.body.appendChild(tr);
    });

    it("Cell comp renders", function() {
        render(<Cell />, { container: row });
    });

    it("snapshot not isLit", function() {
        const { asFragment } = render(<Cell />, { container: row });
        expect(asFragment()).toMatchSnapshot();
    });

    it("snapshot isLit", function() {
        const { asFragment } = render(<Cell isLit />, { container: row });
        expect(asFragment()).toMatchSnapshot();
    });
});
