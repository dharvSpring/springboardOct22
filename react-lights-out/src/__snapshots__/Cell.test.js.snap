// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell comp tests snapshot isLit 1`] = `<DocumentFragment />`;

exports[`Cell comp tests snapshot not isLit 1`] = `<DocumentFragment />`;
